neqo-transport/tests/server.rs

```diff
@@ -939,3 +941,76 @@ fn closed() {
     let res = server.process(None, now() + Duration::from_secs(60));
     assert_eq!(res, Output::None);
 }
+
+#[test]
+fn max_streams() {
+    const MAX_STREAMS: u64 = 100;
+    let mut server = Server::new(
+        now(),
+        test_fixture::DEFAULT_KEYS,
+        test_fixture::DEFAULT_ALPN,
+        test_fixture::anti_replay(),
+        Box::new(AllowZeroRtt {}),
+        Rc::new(RefCell::new(FixedConnectionIdManager::new(9))),
+        ConnectionParameters::default()
+            .max_streams_bidi(MAX_STREAMS)
```

Comment:
Would it be better to have this use StreamType?  That is, 

        ConnectionParameters::default().max_streams(StreamType:BiDi, MAX_STREAMS)